content/en/news/release3.4.md

```
@@ -1,56 +1,74 @@
 
-We are proud to announce the release of OpenMS 3.4.1. Grab it <a href="https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/3.4">here</a>
+We are proud to announce the release of OpenMS 3.4.1. Grab it <a href="https://github.com/OpenMS/OpenMS/releases/tag/release%2F3.4.1">here</a>
 
 In the following you find all important changes to the previous version:
 
+------------------------------------------------------------------------------------------
+----                                OpenMS 3.4.1     (May 2025)                       ----
+------------------------------------------------------------------------------------------
+
+Documentation:
+  - TOPPAS: tutorial update (#8025)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)